### PR TITLE
Update theme with elevations and constants where appropriate. #353.

### LINKED ISSUE #353: Update theme with elevations and constants where appropriate

### PR DESCRIPTION
**Pull request by frano-m**
### Ticket
Closes #353.

### Reviewers
@MillenniumFalconMechanic .

### Changes
- Modified theme with elevations and colours as constants.
- Updated header with elevation as prop.

### CODE REVIEW

File: explorer/app/components/CheckboxMenu/checkboxMenu.styles.ts [hunk 1]
```diff
@@ -2,6 +2,10 @@ import styled from "@emotion/styled";
 import { Menu } from "@mui/material";
 
 export const CheckboxMenu = styled(Menu)`
+  .MuiPaper-menu {
+    margin: 8px 0;
```
Comment (frano-m) [inline explorer/app/components/CheckboxMenu/checkboxMenu.styles.ts:6]: Moved margin to Checkbox styles - margin changes depending on use of `Menu`.

File: explorer/app/components/Layout/components/Sidebar/sidebar.styles.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import styled from "@emotion/styled";
 
 export const Sidebar = styled.div`
+  align-self: stretch;
```
Comment (frano-m) [inline explorer/app/components/Layout/components/Sidebar/sidebar.styles.ts:4]: Required fix; I had main as a grid, and with the switch to flex this is required to stretch the sidebar to full height of `main`.

File: explorer/app/components/common/Button/button.styles.ts [hunk 3]
```diff
@@ -5,7 +5,7 @@ import styled from "@emotion/styled";
 export const ButtonSecondary = styled(Button)`
   background-color: ${({ theme }) => theme.palette.common.white};
   box-shadow: inset 0 0 0 1px ${({ theme }) => theme.palette.smokeDark},
-    0px 1px 0px 0px rgba(0, 0, 0, 0.08);
+    0 1px 0 0 rgba(0, 0, 0, 0.08);
```
Comment (frano-m) [inline explorer/app/components/common/Button/button.styles.ts:8]: Linting.

File: explorer/app/components/Layout/components/Sidebar/sidebar.styles.ts (hunk 2 above)
Comment (MillenniumFalconMechanic) [reply explorer/app/components/Layout/components/Sidebar/sidebar.styles.ts:4]: Thank you!

**Review by MillenniumFalconMechanic — APPROVED** [review]
LGTM, thanks Fran!

**Pull request merged**